develop/smart-contracts/evm/native-evm-contracts.md

```diff
@@ -1,4 +1,98 @@
+- [**PolkaVM**](https://github.com/paritytech/polkavm){target=\_blank} - a custom virtual machine replacing EVM, featuring a RISC-V-based register architecture for efficient hardware translation and 64-bit word size for faster arithmetic. Includes an interpreter for lightweight tasks and plans for a JIT compiler, allowing Solidity to interact with high-performance languages
+- [**Revive**](https://github.com/paritytech/revive){target=\_blank} - compiles Solidity for PolkaVM by translating [YUL](https://docs.soliditylang.org/en/latest/yul.html){target=\_blank}, the intermediate output from the solc compiler, to RISC-V. This approach simplifies development and ensures full compatibility with all Solidity versions and features
+- [**Revive Remix**](https://github.com/paritytech/revive-remix){target=\_blank} - a modified fork of Remix adapted to use a backend compiler instead of an in-browser one, accommodating the LLVM-based Revive compiler's complexity
+
+```mermaid
+graph TD
+    A[pallet_revive] -->|Executes contracts| B[PolkaVM]
+    A -->|Compiles Solidity| C[Revive]
+    C -->|Compiles through| D[Revive Remix]
+    B -->|Handles transactions| E[Ethereum-style Transactions Handling]
+    E --> F[Transaction Conversion & Dispatch]
+    B -->|Executes Solidity code| G[Solidity Execution]
+```
+
+### PolkaVM: Key Performance and Compatibility Differences
+
+PolkaVM offers significant performance improvements over existing Polkadot smart contract execution environments, such as Frontier, thanks to its custom architecture based on RISC-V and 64-bit word size. This approach enables more efficient hardware translation and faster arithmetic, allowing additional languages like C or Rust to be integrated for further performance optimization. Unlike Ethereum, which uses a stack-based EVM, PolkaVM’s register-based design allows for faster compilation times and better compatibility with modern hardware. However, PolkaVM’s focus on performance means it deviates from Ethereum in key areas, such as gas modeling and contract execution.
+
+PolkaVM introduces a multi-dimensional gas model to address these differences, incorporating resources like computation time, proof size, and storage deposits. This provides a more accurate representation of contract execution costs and reduces overcharging for memory allocations. PolkaVM also avoids Ethereum’s approach of using a single resource for gas, making it possible to meter resources at the transaction level and for cross-contract calls. For compatibility, PolkaVM’s RPC interface closely mirrors Ethereum’s, with minor adjustments needed for some operations. Additionally, PolkaVM hides the existential deposit requirement to ensure smooth operation across networks, ensuring users aren’t surprised by the underlying balance limitations.
+
+## Get Started
+
+### Connect to Asset Hub
+
+Install the MetaMask browser extension and add a custom network using the settings provided below:
```

Comment:
Maybe you can say something like: Install any EVM compatible wallet like metamask